src/components/icons/LensIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import { SvgIcon } from '@mui/material';
+
+interface IconProps {
+  color: string;
+}
+
+export const LensIcon = ({ color }: IconProps) => {
+  return (
+    <SvgIcon xmlns="http://www.w3.org/2000/svg" aria-labelledby="title">
+      <title>Lens icon</title>
+      <svg
+        width="25"
+        height="25"
+        viewBox="0 0 96 61"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
```

Comment:
These attributes should be `fillRule` and `clipRule`

Reply:
good catch `437b2fb12fc1c1d69175eb024d951509eb2c887a`